### PR TITLE
Switch to PHP 7.1+

### PR DESCRIPTION
**Pull request by JeroenDeDauw**
Includes https://github.com/wmde/github-webhooks/pull/14

Since composer.lock makes it difficult to parallelize without creating merge conflicts

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,21 +2,19 @@ language: php
 
 matrix:
   include:
-  - php: 7
+  - php: 7.1
```
Comment (wiese) [inline .travis.yml:5]: You want to switch to 7.1 only or would it be worthwhile to prove 7.0 is working as well?

File: phpcs.xml [hunk 2]
```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0"?>
+<!--
+	- https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml
+	- https://github.com/squizlabs/PHP_CodeSniffer/tree/master/CodeSniffer/Standards
+-->
+<ruleset name="PsrLogTestDoubles">
```
Comment (wiese) [inline phpcs.xml]: `name="github-webhooks"` ?

File: composer.json [hunk 3]
```diff
@@ -12,7 +12,10 @@
     "require-dev": {
         "phpunit/phpunit": "^6.1",
         "wmde/psr-log-test-doubles": "^2.1",
-        "symfony/browser-kit": "^3.1"
+        "symfony/browser-kit": "^3.1",
+
+        "squizlabs/php_codesniffer": "~2.5",
```
Comment (wiese) [inline composer.json]: Wouldn't a current version be more appropriate? 2.9+

File: composer.json [hunk 4]
```diff
@@ -12,7 +12,10 @@
...
+        "mediawiki/mediawiki-codesniffer": "~0.6.0"
```
Comment (wiese) [inline composer.json]: Why not use the 0.8.0 with dependency to squizlabs/php_codesniffer 2.9?

File: .travis.yml (hunk 1 above)
Comment (JeroenDeDauw) [reply .travis.yml:5]: I switched composer.json to ~7.1, so can't list 7.0 here. Doing this in dedicated PR so I can more easily use 7.1 features afterwards without complicating PRs about other things.

File: composer.json (hunk 4 above)
Comment (JeroenDeDauw) [reply composer.json]: Sure, good idea if that works. I copied this file from another component.

**Comment by wiese** [comment]
Great. You want to merge that into master or the "phpcs" branch (#14)?

**Pull request merged**